tests/test_evaluations/test_evaluation_functions.py

```diff
@@ -28,7 +28,7 @@ def _check_list_evaluation_setups(self, **kwargs):
         # Check if output and order of list_evaluations is preserved
         self.assertSequenceEqual(evals_setups["run_id"].tolist(), evals["run_id"].tolist())
         # Check if the hyper-parameter column is as accurate and flow_id
-        for index, row in evals_setups.iterrows():
+        for index, row in evals_setups.head(3).iterrows():
```

Comment:
Do you think this could be parametrized with slow/fast as discussed offline this morning?